packages/inter-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -51,7 +51,7 @@ const trace = makeTracer('LiqI', false);
  *   priceAuthority: PriceAuthority,
  *   reservePublicFacet: AssetReservePublicFacet,
  *   timerService: TimerService,
- *   debtBrand: Brand,
+ *   debtBrand: Brand<'nat'>,
```

Comment:
Doesn't `Brand<K>` default to `Brand<'nat'>`? Why do we need to change this in so many places?

Please add the justification to the "nat amounts" commit message and/or make a comment in the ERTP type declaration(s).

Reply:
> Doesn't Brand<K> default to Brand<'nat'>?

That'd be nice. But nope, https://github.com/Agoric/agoric-sdk/blob/master/packages/ERTP/src/types-ambient.js#L87-L89

> Why do we need to change this in so many places?

Because by these `Brand` declarations they're claiming that they can be any AssetKind.

Reply:
OK... I wonder about a `FungibleBrand` alias... for another issue...